static/js/rendered_markdown.js

```diff
@@ -166,6 +166,26 @@ export const update_elements = (content) => {
         }
     });
 
+    content.find("h4.poll-question-header").each(function () {
+        if ($(this).text().includes("<time:")) {
+            const text = $(this)
+                .text()
+                .match(/<time:(.*)>/)
```

Comment:
Any parser that involves regexes and `.*` isn’t remotely careful enough. (For example, this will fail spectacularly on a message with two time mentions.)

Reply:
Yes I have encountered the bug you mentioned. What exactly is expected from a JS-side parser? @andersk

Reply:
I have fixed this in the below commits.